platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/AttributeSharingHashSessionIdManager.java

```diff
@@ -82,10 +81,15 @@ public void invalidateSession(
 
       final Optional<String> sessionIdOptional =
           idManager
-              .latestSessionAttributes
-              .entrySet()
+              .dataStores
```

Comment:
❗️ the `dataStores` list is being accessed from multiple threads either by iteration (as here) or to add or delete entries. However, I do not see any concurrency considerations which would require the collection to be synchronized on whenever accessed, modified or while being iterated.